Server/src/main.py

```
@@ -8,6 +8,32 @@
+        "Optional",
+        "Dict",
+        "List",
+        "Tuple",
+        "Set",
+        "FrozenSet",
+    )
+    for _name in _typing_names:
+        if not hasattr(builtins, _name) and hasattr(_typing, _name):
+            setattr(builtins, _name, getattr(_typing, _name))  # type: ignore[attr-defined]
+except Exception:
+    pass
```

Ruff (0.14.10)

`try`-`except`-`pass` detected, consider logging the exception

(S110)